### DISCUSSION TITLE
how to download the lighthouse deck ?

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1594] [General]
Hi,

how to download the lighthouse deck ? My FPGA was burnt out, so I replaced it with a new FPGA and Flash chip

**Comment by ataffanel** [comment]
You likely only killed the FPGA, memories are much harder to burn, so I would suggest to solder back your memory which will get your deck up and running directly.

**Reply by justinleeyang** [reply]
I haved tryed, but no start up, so use FT232H cable to download the memories ?

**Reply by justinleeyang** [reply]
Must use Lattice's HW-USB-2A or HW-USBN-2B programer to download the bin file ?

**Reply by justinleeyang** [reply]
I haved download new fpga and spi flash, but crazyflie2.1 start up, report as following, why ?:

```
LHFL: Cannot communicate with lighthouse bootloader, aborting!
LH: FPGA not booted. Lighthouse disabled!
```

![image](https://github.com/user-attachments/assets/7c59e784-2f3e-422b-a65c-42e9714923c8)
![image](https://github.com/user-attachments/assets/1e7478ca-b496-424b-a121-412e73225060)
![image](https://github.com/user-attachments/assets/6f9425b5-b1ee-4c16-abe5-e2d45398aa75)

**Reply by justinleeyang** [reply]
run successfully, thanks
![image](https://github.com/user-attachments/assets/d33eaf3b-29ae-444e-8cb6-dff87878dd94)
